### PR TITLE
Test webpack fixes

### PR DESCRIPTION
**Pull request by vespasianvs**

### CODE REVIEW

File: src/parser/tag.ts [hunk 1]
```diff
@@ -308,6 +308,7 @@ const getStringValue = (rawData: DataView, charset?: Charset, vr?: string) => {
 
 const getSingleStringValue = (
 	rawData: DataView,
+	// eslint-disable-next-line default-param-last
```
Comment (nickhingston) [inline src/parser/tag.ts:311]: instead of this make the rest of the params = undefined / null?

File: src/parser/index.test.js [hunk 2]
```diff
@@ -181,7 +181,7 @@ describe("parser tests", () => {
 		expect(image.tags).toBeTruthy();
 		// console.log(toJSONString(image.tags));
 		// fileNameToTagHash[key] = shaFromJSON(image.tags);
-		expect(shaFromJSON(image.tags)).toEqual("c18190989e0b6fc58c19287507d4074c46b45c35");
+		expect(shaFromJSON(image.tags)).toBe("c18190989e0b6fc58c19287507d4074c46b45c35");
```
Comment (nickhingston) [inline src/parser/index.test.js:184]: we not gonna have same problem here?

**Review by nickhingston — CHANGES_REQUESTED** [review]
removing the .dcm files - do the examples still work?

File: src/parser/index.test.js (hunk 2 above)
Comment (vespasianvs) [reply src/parser/index.test.js:184]: Oh...missed those cos they were passing...hmmm....

**Review by nickhingston — APPROVED** [review]

**Pull request merged**